ethereum/block/wait_for_receipt.go

```diff
@@ -42,16 +42,11 @@ func WaitForReceipt(ctx context.Context, ec *ethclient.Client, txHash ethcommon.
 			continue
 		}
 		if receipt.Status != ethtypes.ReceiptStatusSuccessful {
-			err = fmt.Errorf("transaction failed (gas-lost=%d tx=%s block=%d), %w",
-				receipt.GasUsed, txHash, receipt.BlockNumber, ErrorFromBlock(ctx, ec, receipt))
+			err = fmt.Errorf("mined transaction failed (%s): %w",
```

Comment:
Hopefully, the verb "mined" is still used even though Ethereum is now PoS.

Reply:
I'd change this to "transaction was included but failed" or something like that

Reply:
Done.